Problems/1629/ans_1629.py

```diff
@@ -0,0 +1,17 @@
+#https://www.acmicpc.net/problem/1629
+# (A^B)%C 로 하게 되면 시간 복잡도가 올라가서 분할로 접근해야 함
+
+A, B, C = list(map(int, input().split()))
+
+def func(a, b, c):
+  if b == 1:
+    return a % c
+
+  n = func(a, b//2, c)
+  if b%2 :
```

Comment:
```suggestion
  if b%2 == 1:
```
이게 좀 더 명백하지 않을까? b%2의 결과가 flag로서의 1이 아니라, modular 계산으로서의 1이니..